IBAnimatable/FlipAnimator.swift

```diff
@@ -21,14 +21,13 @@ public class FlipAnimator: NSObject, AnimatedTransitioning {
   fileprivate var horizontal: Bool = false
 
   // MARK: - Life cycle
-  public init(from direction: TransitionAnimationType.Direction, transitionDuration: Duration) {
+  public init(from direction: TransitionAnimationType.Direction, duration: Duration) {
```

Comment:
I don't think you should add an entry for the change of self. since it's only internal changes, but you should add an entry in the API breaking change to explain the prototype change transitionDuration -> duration because that may break for the users.